Models/MCTS_Util.py

```diff
@@ -6,27 +6,52 @@
 
 def create_default_mapping():
     mappings = []
+    mask = []
 
-    for a in range(38):
-        for b in range(37):
+    for b in range(37):
+        for a in range(38):
             if a == 37:
                 mappings.append(f"4_{b}")
+                mask.append(1)
             else:
                 mappings.append(f"5_{a}_{b}")
+                if a == b:
+                    mask.append(0)
+                else:
+                    mask.append(1)
 
-        for b in range(10):
+    for b in range(10):
```

Comment:
And bench size

Reply:
I'll move from Simulator config to main config.